pkg/hub/controllers/slicegateway_controller.go

```diff
@@ -50,6 +50,7 @@ type SliceGwReconciler struct {
 }
 
 var sliceGWController = "workersliceGWController"
+var sliceGwFinalizer = "controller.kubeslice.io/hubSpokeSliceGw-finalizer"
```

Comment:
worker.kubeslice.io/sliceGw-finalizer

Reply:
I am following the same naming convention like slice controller at hub side has.
I can remove the hubSpoke after slash but the prefix should be controller IMO to make it consistent